cmd/generate.go

```diff
@@ -104,10 +104,11 @@ func (gc *generateCmd) validate(cmd *cobra.Command, args []string) error {
 
 func (gc *generateCmd) mergeAPIModel() error {
 	var err error
-
+	log.Debugln("mergeAPIModel")
```

Comment:
Are these log statements meant to be a part of the final PR?

Reply:
nope, they are not. just pushed a fix. thanks!